app/views/playlist_items/_current_item.html.erb

```diff
@@ -17,7 +17,7 @@ Unless required by applicable law or agreed to in writing, software distributed
 <% @current_clip = AvalonClip.find(@current_playlist_item.clip_id) %>
 <% @current_masterfile = MasterFile.find(@current_playlist_item.clip.source.split('/').last) %>
 <% @current_mediaobject = MediaObject.find(@current_masterfile.media_object_id) %>
-<section id="metadata">
+<section id="source_details">
```

Comment:
This partial is now only used by mejs2. I wonder if changing this id will break mejs2 stuff.

Reply:
It looks like it is still working from a quick test locally.